nest/src/AggregateToken.sol

```diff
@@ -381,12 +381,6 @@ contract AggregateToken is ComponentToken, IAggregateToken, ERC1155Holder {
             revert AssetNotSupported();
         }
```

Comment:
Let's remove all this stuff too. If this is a NestTeller shouldn't we do:

INestTeller teller = INestTeller(_teller);

or

IComponentToken teller = IComponentToken(_teller);